include/aquamarine/output/Output.hpp

```diff
@@ -88,6 +91,8 @@ namespace Aquamarine {
         void                  enableExplicitOutFenceForNextCommit();
         void                  resetExplicitFences();
         void                  setCTM(const Hyprutils::Math::Mat3x3& ctm);
+        void                  setWideColorGamut(bool wcg);
+        void                  setHDRMetadata(hdr_output_metadata& metadata);
```

Comment:
please wrap this into a struct with a library-side ctor, so that we can add more stuff to it in the future w/o abi breaks and too many funcs

Reply:
Those funcs can be used separately and there is not much else AQ can do regarding HDR. `hdr_output_metadata` is from `libdrm/drm_mode.h` and I don't see a point in creating yet another wrapper around it. At least not until ICC and other colorimetry stuff is ready.

Reply:
yes but then when you add ICC you need yet another thing. Do we really want that?

Reply:
This should be everything needed for HDR. ICC stuff shouldn't need any changes to this API and probably will be entirely in HL or hyprgraphics. At least I don't see any other useful drm properties regarding HDR or ICC.

Reply:
ok edoek